addons/project/models/project.py

```diff
@@ -2004,8 +2004,9 @@ def create(self, vals_list):
                 vals['date_assign'] = fields.Datetime.now()
                 if not project_id:
                     user_ids = self._fields['user_ids'].convert_to_cache(vals.get('user_ids', []), self)
-                    if self.env.user.id not in user_ids:
-                        vals['user_ids'] = [Command.set(list(user_ids) + [self.env.user.id])]
+                    currenlt_user = [] if self.env.ref('base.user_root').id == self.env.user.id else [self.env.user.id]
```

Comment:
This should be done before looping.
```suggestion
                    current_user = [] if self.env.ref('base.user_root').id == self._uid else [self._uid]
```

Reply:
Or better
```suggestion
import SUPERUSER_ID

        is_superuser = self._uid == SUPERUSER_ID
        for vals in vals_list:
            ...
                    if not is_superuser and currenlt_user not in user_ids:
                        vals['user_ids'] = [Command.set(list(user_ids) + currenlt_user)]
```